client/src/components/Header.tsx

```diff
@@ -4,7 +4,8 @@ import { Moon } from '../icons/Moon';
 
 interface HeaderProps {
   deleteUserHandler?: () => void;
```

Comment:
Can remove this from the interface and youse `showDeleteUserModal` in the jsx below